### DISCUSSION TITLE
VPB --reprojectsources  use in script

### DISCUSSION THREAD

**Discussion by SeamusAlba** [openscenegraph/OpenSceneGraph#1265] [General]
Hello,  New to posting on the forum.  I am making a script to build a .ive database.  I am looking at using the --reprojectsources.  Is the code script below the proper way to call my image for --reprojectsources.   I have other questions, but that is another discussion.   Thanks

``
vpbmaster -d ConvertedNed -t ProjectedImages --ReprojectSources 3210634_ne.tif --tile-image-size 512 -v 1.00 -e -106.0 32.0 1.00 1.00 -v 1.00 -o DonaAna\DonaAna.ive -a DonaAna\DonaAna.osga

**Comment by robertosfield** [comment]
You'll need to look at the source to figure out how this old software works under the hood as I can't recall the details off the top of my head, and I'm not about to dive back into the code base to answer questions as I simply don't have the time for this level of support on software I haven't worked on or used for many years.

What I can say is it's rather odd that you're using .ive when this format was deprecated by the .osgb format back in in 2011.  I can only presume you're using very old examples/documentation.

**Comment by SeamusAlba** [comment]
Thank you for the reply!   We still use a very good 3D real time display
for our data.  I am rebuilding a GUI that will automatically take the users
Elevation and imagery and generate a 3D terrain database.  Until we change
the 3D display format, I am stuck with .IVE.   I am going on memory as
well, as I have no old examples of how to use the --reprojectsources.  I
will try it and see what it does.  I am thinking of submiting to OSG with
all source for anyone to use. Thanks for your help.

**Comment by SeamusAlba** [comment]
Bob,

Where can I get the last known stable Virtual Planet Builder
(VPBmaster.exe) ? It used to be located on
http://virtualplanetbuilder.osgforge.org/. I can't seem to find it on the
OSG site.

r/
Jim Northrup

**Reply by robertosfield** [reply]
Who's Bob?

I haven't personally ever built Virtual Planet Builder executables, let alone Windows ones as very rarely used Windows.  If you want to use VirtualPlanetBuilder you'll need to build it from source and accept that it's very old no longer supported software, it remains open source so you are free to use it if you still find it useful.
